### PR TITLE
Various build script touchups

### PR DESCRIPTION
**Pull request by Hoverbear**
@workingjubilee can you take a look?

### CODE REVIEW

**Comment by workingjubilee** [comment]
So, we will host multiple target.json examples soon enough (for aarch64 if nothing else), with different names, so I would prefer fixing the name to `x86_64-postgres-linux-gnu.json` (so it matches the appropriate build tuple and so we can parameterize the script).

**Comment by workingjubilee** [comment]
Cool, I think this works on this end. Thank you!

**Comment by Hoverbear** [comment]
Yay!

File: build [hunk 1]
```diff
@@ -6,15 +9,16 @@ CARGO_PROFILE_RELEASE_DEBUG_ASSERTIONS=true \
...
 	--target "${BUILD_THIS}" \
     -Zbinary-dep-depinfo \
     --release \
     --features "compiler-builtins-mem" \
     --manifest-path "library/test/Cargo.toml"
 
 if [[ ${1} == "copy" ]]; then
-    cp target/${BUILD_THIS}/release/deps/*.rlib $(rustc --print sysroot)/lib/rustlib/${BUILD_THIS}
+    mkdir -p $(rustc --print sysroot)/lib/rustlib/${BUILD_THIS}/lib
+    cp --force --verbose target/${BUILD_THIS}/release/deps/*.rlib $(rustc --print sysroot)/lib/rustlib/${BUILD_THIS}/lib
```
Comment (workingjubilee) [inline build:21]: Thank you for figuring these ones out.

**Review by workingjubilee — APPROVED** [review]

**Pull request merged**